### ISSUE TITLE
Publish SemVer tags to luarocks.org?

### ISSUE THREAD

**Issue by mrcjkb** [nushell/tree-sitter-nu#103]
Hey :wave: 

[rocks.nvim supports using tree-sitter parsers that have been published to luarocks.org as plugin dependencies](https://github.com/nvim-neorocks/rocks.nvim?tab=readme-ov-file#deciduous_tree-enhanced-tree-sitter-support).

This means that rocks.nvim users can just `:Rocks install my-nushell-plugin.nvim`, and the tree-sitter parser will be installed automatically, without having to use nvim-treesitter to install parsers.

For parsers that are supported by nvim-treesitter, we have an automated workflow to package/publish them using [the NURR](https://github.com/nvim-neorocks/nurr).
But for parsers that aren't natively supported by nvim-treesitter, it's better if the authors manage them themselves.

Some examples:

- [tree-sitter-norg](https://luarocks.org/modules/neorg/tree-sitter-norg)
- [tree-sitter-orgmode](https://luarocks.org/modules/kristijanhusak/tree-sitter-orgmode).

Would you be open to publishing SemVer releases (e.g. using the release-please github action) and pubishing `tree-sitter-nu` to luarocks.org?
If so, I'd be happy to set up the GitHub actions for you.

**Comment by fdncred** [comment]
👋🏻 

We're of two minds in regard to publishing.
1. We're happy to have tree-sitter-nu in as many places as possible for users to easily install and use it.
2. It ends up being a big pain to support because people change things and then publishing breaks without anyone who cares to support it.

**Comment by mrcjkb** [comment]
Thanks for the feedback. 
In that case I suppose it's better for you if we add it to the NURR.

**Comment by mrcjkb** [comment]
I have [added tree-sitter-nu to the NURR](https://github.com/nvim-neorocks/nurr/commit/e1a3e5f6ff87c5f502210551193c1c0a813432b4), so a `scm` version is [now available on luarocks.org](https://luarocks.org/modules/neorocks/tree-sitter-nu).

It would be nice to have semver releases so that users can pin versions.
Are there any plans for that?

**Comment by fdncred** [comment]
Yes, but probably not until we have a maintainer passionate enough about tree-sitter-nu to finish the grammar and highlighting and do releases.
